patches/chrome-browser-ui-views-location_bar-location_bar_view.cc.patch

```diff
@@ -0,0 +1,25 @@
+diff --git a/chrome/browser/ui/views/location_bar/location_bar_view.cc b/chrome/browser/ui/views/location_bar/location_bar_view.cc
+index ec6eafd7bd7d471a74cec798b993446d99b09703..1b8cc4551c7f2c3757cbb214446f157ce581781c 100644
+--- a/chrome/browser/ui/views/location_bar/location_bar_view.cc
++++ b/chrome/browser/ui/views/location_bar/location_bar_view.cc
+@@ -464,7 +464,7 @@ gfx::Size LocationBarView::CalculatePreferredSize() const {
+   return min_size;
+ }
+ 
+-void LocationBarView::Layout() {
++void LocationBarView::Layout(views::View* right_most) {
+   if (!IsInitialized())
+     return;
+ 
+@@ -563,6 +563,11 @@ void LocationBarView::Layout() {
+     }
+   };
+ 
++  if (right_most && right_most->visible())
++    trailing_decorations.AddDecoration(0,
++                              location_height + (vertical_padding * 2),
```

Comment:
how about using `height()` instead of `location_height + (vertical_padding * 2)`?
I think both are same.